### DISCUSSION TITLE
logLevel: debug does not appear to have any effect?

### DISCUSSION THREAD

**Discussion by gdevenyi** [requarks/wiki#5065] [Error / Bug Report]
In attempting to debug LDAP not working, I enabled the "LDAP Debug" flag in the developer page and nothing happened. So I changed to  `logLevel: debug` in the config, surprisingly, still nothing happened in the logs. I would've expected far more info like the database connection details etc, but it seems the these debug settings have no effect.

This code was very recently changed, possibly related
https://github.com/Requarks/wiki/commit/2815f38c52c431bab0f53c2b1a6f9b2f3e2afcbf

Trying an older version, the recent logging change didn't cause this.

**Comment by kingfisher77** [comment]
Still do not see debug output, it is always info no matter was I set as logLevel.

Docker Wiki.js 2.5.292

**Comment by DavidOhsieAtTurbo** [comment]
I believe that the answer to this question is that very little debug level logging is done in Wiki.js.  Grepping through the source code gives only a few places were debug logging is enabled.

```
find . -type f -exec grep --color=auto -nH --null -e logger.debug \{\} +
./server/core/letsencrypt.js44:            WIKI.logger.debug(`${ev}: ${JSON.stringify(msg)}`)
./server/core/db.js237:        WIKI.logger.debug(ev)
./server/models/users.js528:      WIKI.logger.debug(`Password reset attempt on nonexistant local account ${email}: [DISCARDED]`)
./server/models/users.js899:      WIKI.logger.debug(`Updating user ${userId} avatar data...`)
```

**Comment by pgi-jsanchez** [comment]
I tested some debug calls at the authentication.js for Google and the output isn't there at all. Tried with the environment variable LOG_LEVEL at the container, force changed the config.yml directly and difference. So I can confirm that WIKI.logger.debug does not work properly. 

Despite that is a good to have more verbose messages when things goes wrong, in cases that is required to look closer to the code and certain issues, and considering how big this project is growing, things should be done.  Plus, I suggest to add into the wiki the LOG_LEVEL variable once there are more debug messages.

WikiJs version: 2.5.296 (modified with debug calls)
Postgres version: 11

Modified file just for test:
![Captura de pantalla-20230209171425-999x411](https://user-images.githubusercontent.com/38993394/217960269-03061569-3100-4765-9449-0f0252b1edb7.png)

Logs from Portainer: 
![Captura de pantalla-20230209171530-897x232](https://user-images.githubusercontent.com/38993394/217960422-3494230e-32d0-4d17-ae2f-8bfe41304e8e.png)
